go/billing/migrations/0024_auto__add_field_statement_status.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Statement.status'
+        db.add_column(u'billing_statement', 'status',
+                      self.gf('django.db.models.fields.CharField')(default='statement_pending', max_length=20),
```

Comment:
I wonder if the default here shouldn't be `statement_completed` since all existing statements are presumably complete?